view/frontend/templates/checkout/success.phtml

```diff
@@ -45,71 +45,92 @@
     </script>
     <div id="ActionContainer"></div>
 <?php endif; ?>
+
 <?php if ($block->showAdyenGiving()):
     $checkoutConfig = /* @noEscape */
     $block->getSerializedCheckoutConfig();
     $scriptString = __('window.checkoutConfig = %1;', $checkoutConfig);
-?>
-<script>
-    <?= /* @noEscape */ $scriptString ?>
-    require([
-        'jquery',
-        'Adyen_Payment/js/adyen',
-        'Adyen_Payment/js/model/adyen-payment-service'
-    ], function ($, AdyenCheckout, adyenPaymentService) {
-    (async function () { // RequireJS does not support async callback
-        function handleOnDonate(state, component) {
-            if (state.isValid) {
-                let payload = state.data;
-                payload.returnUrl = window.location.href;
-
-                const orderId = <?= $block->escapeJs($block->getOrder()->getId()) ?>;
+    ?>
+    <div id="donation-container"></div>
+    <script>
```

Comment:
Could you please use `secureRenderer()` for rendering scripts? Otherwise, we might face with nonce related CSP issues.